k9mail-library/src/main/java/com/fsck/k9/mail/internet/MimeUtility.java

```diff
@@ -1153,4 +1155,12 @@ static boolean isFormatFlowed(String contentType) {
         }
         return false;
     }
+
+    static boolean isDelSp(String contentType){
```

Comment:
This method is used in library module itself so I think it would not be wise to move it to FlowedMessageUtils.